src/components/dialogs/create-case-dialog.js

```diff
@@ -85,7 +85,15 @@ export function CreateCaseDialog({ onClose, open }) {
         })
             .then()
             .catch((message) => {
-                snackbarMessage(message, 'caseCreationError', { name });
+                message.trim().startsWith('422')
```

Comment:
i'd prefer to add a constant with code status 422 ` const UNPROCESSABLE_ENTITY_STATUS = '422'`
then unstead of doing this `message.trim().startsWith('422')`  you can do this unstead `message.includes(UNPROCESSABLE_ENTITY_STATUS) `

Reply:
done